pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1992,6 +1992,20 @@ func parseUnderlayNetworkConfig(appInstance *types.AppInstanceConfig,
 			return appInstance.UnderlayNetworkList[i].IntfOrder <
 				appInstance.UnderlayNetworkList[j].IntfOrder
 		})
+
+	// calculate IfIdx field for interfaces connected to the same network
+	networkIfs := make(map[uuid.UUID]uint32)
```

Comment:
Took a while to understand the logic. I wonder if it would be more clear if this map was called nextIfindexForNetwork?

Reply:
Yes, good point, renamed